style.css

```diff
@@ -0,0 +1,203 @@
+.status-bar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background-color: #3c3a39;
+}
+
+.clock {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: left;
+  letter-spacing: -0.28px;
+  color: #fff;
+  padding-top: 10px;
+  padding-left: 20px;
+  width: 54px;
+  height: 16px;
+}
+
+.status {
+  float: right;
+  width: 66.66px;
+  height: 11.34px;
+  padding-right: 20px;
+}
+
+.name {
+  width: 77px;
+  height: 24px;
+  left: 12px;
+  top: 10px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
+  padding-top: 10px;
+}
+
+.dropdown {
+  float: right;
+  width: 40px;
+  height: 40px;
+  padding-right: 19px;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  background-color: #1c1a19;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
> You have done an amazing job following the design guidelines, but the container is a bit small. Mobile screens can be of different sizes and start from `425 px` (mobile lg screen). While your design is good for devices starting from `375px`(mobile md screen). 
- [ ] To make the portfolio more responsive I recommend increasing the width property to `425 px`
<img width="450" alt="Screenshot 2022-03-17 at 10 10 16" src="https://user-images.githubusercontent.com/77038610/158747669-571d29f7-a18c-4b02-b0a8-ae073ffdc1d0.png">